exporter/clickhouseexporter/exporter_logs_test.go

```diff
@@ -101,6 +105,32 @@ func TestExporter_pushLogsData(t *testing.T) {
 	})
 }
 
+func TestLogsExporter_createDatabase(t *testing.T) {
+	c := &Config{
+		ExporterSettings: config.NewExporterSettings(component.NewID(typeStr)),
+		TimeoutSettings:  exporterhelper.NewDefaultTimeoutSettings(),
+		QueueSettings:    QueueSettings{QueueSize: exporterhelper.NewDefaultQueueSettings().QueueSize},
+		RetrySettings:    exporterhelper.NewDefaultRetrySettings(),
+		DSN:              "tcp://mydatabase-clickhouse-headless:9000/mydatabase",
+		LogsTableName:    "otel_logs",
+		TracesTableName:  "otel_traces",
+		TTLDays:          7,
+	}
+
+	// a part of createDatabase()
+	createFunc := func(cfg *Config) string {
+		database, _ := parseDSNDatabase(cfg.DSN)
+		if database == defaultDatabase {
+			return ""
+		}
+		// use default database to create new database
+		dsnUseDefaultDatabase := fmt.Sprintf("%s%s", cfg.DSN[0:strings.LastIndex(cfg.DSN, database)], defaultDatabase)
```

Comment:
this doesn't actually call the method under test. Please make sure it does, as there is no guarantee your test does anything if the code changes.